### ISSUE TITLE
Custom Instruction for Best Practices and better code proposal

### ISSUE THREAD

**Issue by RobinDelage** [microsoft/vscode-copilot-release#383]
The point would be to give custom instruction (like the new version of Chat GPT) to be sure that Copilot make propositions based on our way of coding and best pratices.

You would be able to explain the contexte of the application ou the best practices you have for specifiques languages

**Comment by Elie77350** [comment]
Hello, where is the project at?
I've just joined this role, and I have an account with GPT-4. I've been testing this feature for several months. This option should be available on GitHub Copilot because it's very powerful!
This feature saves a LOT of time in development. I mainly develop with Vue 3.4 Composition API and <script setup>, and I've trained GPT-4 to operate with all the latest Vue best practice guidelines and standards, and the results are quite satisfying.
What's unfortunate is the issue with obsolescence.

Example: I recently used the "ZOD" library for my project, and Visual Studio Code sent me an error message to let me know that "toFormValidator" and "toFieldValidator" are deprecated. I then queried GPT-4, which didn't know because of its training data cut-off in 2021.
I then checked with GitHub Copilot, which confirmed and suggested the "toTypedSchema" alternative instead.
